ports/esp32/Makefile

```diff
@@ -354,6 +354,9 @@ SRC_C = \
 	$(wildcard $(BOARD_DIR)/*.c) \
 	$(SRC_MOD)
 
+SRC_CPP += \
```

Comment:
Should this be `SRC_CXX` because (1) it matches `CXXFLAGS`; (2) it won't get confused with $(CPP) which is the preprocessor?  I'm not sure what the general convention is with C++ make variables...

Reply:
Me neither, I've seen all kinds of things; but using `CXXFLAGS` is very common so make sense using `CXX` everywhere is for consistency.

Note there are also Makefiles which use one single SRC then split it in %.c and %.cpp. We could do that, but it's a bit strange to have it called SRC_C and then add .cpp files to it? Plus I tried that at one point and quickly ran into problems, don't remember what eactly though. But if you think that's better in the end I can look into it.

Reply:
I think having separate `SRC_C` and `SRC_CXX` is fine.